src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -590,7 +590,7 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         /// <param name="maxCalibrationExamples">Number of instances to train the calibrator.</param>
         /// <param name="useProbabilities">Use probabilities (vs. raw outputs) to identify top-score category.</param>
         /// <typeparam name="TModel">The type of the model. This type parameter will usually be inferred automatically from <paramref name="binaryEstimator"/>.</typeparam>
-        public static Ova OneVersusAll<TModel>(this MulticlassClassificationCatalog.MulticlassClassificationTrainers catalog,
+        public static OneVsAllTrainer OneVersusAll<TModel>(this MulticlassClassificationCatalog.MulticlassClassificationTrainers catalog,
             ITrainerEstimator<ISingleFeaturePredictionTransformer<TModel>, TModel> binaryEstimator,
             string labelColumnName = DefaultColumnNames.Label,
             bool imputeMissingLabelsAsNegative = false,
```

Comment:
Why do we need label and not features here?

Reply:
I think this label column is purely for calibration. BinaryEstimator contains point on label and feature column.

---
In reply to: [260486559](https://github.com/dotnet/machinelearning/pull/2743#discussion_r260486559) [](ancestors = 260486559)

Reply:
Makes sense! Thank you @Ivanidzo4ka